sc/social/like/subscribers.py

```diff
@@ -107,3 +114,50 @@ def assign_canonical_url(obj, event):
             'Canonical domain not set in Social Media configlet; '
             "Facebook's Open Graph canonical URL (og:orl) will not be available"
         )
+
+
+def social_content_check(obj, event):
+    """Check if content is valid if enabled in sc.social.content, in action when edit."""
+
+    request = getRequest()
+
+    try:
+        view = getMultiAdapter((obj, request), name='social_likes_view')
+    except ComponentLookupError:
+        logger.info('not instaled.')
+        return
+
+    if view.enabled:
+
+        title = getattr(obj, 'title', None)
+        v_title = validate_title_social(title)
+        if title and v_title:
+            msg = v_title
+            api.portal.show_message(message=msg, request=request, type='warning')
+
+        try:
+            description = obj.Description()
+            v_description = validate_description_social(description)
+            if description and v_description:
+                msg = v_description
+                api.portal.show_message(message=msg, request=request, type='warning')
+        except AttributeError:
+            pass
+
+        image = get_content_image(obj)
+        if image:
+            v_image = validate_image_social(image)
+            if v_image:
+                msg = v_image
+                logger.info(msg)
```

Comment:
Why do you use `logger.info` here but no in the `title` and `description` checks above?

Reply:
I prefer `logger.debug()`